governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals/utils.ts

```diff
@@ -117,3 +118,24 @@ const getTransactionSummary = (instruction: MultisigInstruction) => {
       } as const
   }
 }
+
+export const isSolanaPullProgram = (programId: PublicKey) => {
+  const solanaPullProgramIds = [
+    new PublicKey('rec5EKMGg6MxZYaMdyBfgwp4d5rB9T1VQH5pJv5LtFJ'),
+    new PublicKey('HDwcJBJXjL9FpJ7UBsYBtaDjsBUhuLCUYoz3zr8SWWaQ'),
+    new PublicKey('pythWSnswVUd12oZpeFP8e9CVaEqJg25g1Vtc2biRsT'),
+  ]
```

Comment:
hardcoding here because we only need the program ids and installing the `@pythnetwork/pyth-solana-receiver` package seems overkill + the program ids probably wont change that often

Reply:
To be fair since it's the same monorepo I think it's fine